changelog/12795.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature

```

Comment:
We use a particular changelog format for new features. I _think_ this is more of an improvement, so I moved it to that section. @cipherboy I'm going to tag you on an enterprise PR if you don't mind giving it a look.